### ISSUE TITLE
If the password contains only numbers, an error will occur

### ISSUE THREAD

**Issue by k1asa** [danielperna84/hass-configurator#113]
If the password contains only numbers, an error will occur.

```
if PASSWORD and PASSWORD.startswith("{sha256}"):
AttributeError: 'int' object has no attribute 'startswith'
```

I use configurator on hassio in raspbian.

**Comment by danielperna84** [comment]
This has been fixed in the 0.3.2 release. There always is a delay between the release of the standalone version and the hassio add-on. In a couple of days the upgrade will be available for you.

**Comment by k1asa** [comment]
Thanks a lot!!
